submissions/Mifaresss/js_dom/script.js

```diff
@@ -0,0 +1,143 @@
+'use strict'
+
+function hiddenHomeScreen(event) {
+	if (event.target.tagName !== 'BUTTON') return false;

```

Comment:
You have to practice how this change

Reply:
is this bad practice?

Reply:
Fixed

Reply:
No, but if you change the tag element from button to link. Your functionality will not work.
Then we leave these changes, but try not to do this in the future.

Reply:
ok, i get it)